12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,26 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
-
+echo [a-z]*
   2. All files whose name starts with an uppercase letter
   Answer:
-
+echo [A-Z]*
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
-
+echo *[a,e,i,o,u]
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
-
+echo *[!a,!e,!i,!o,!u]
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
-
+cd src
+echo *.c
   6. All files with extension ".c" under any folder
   Answer:
-
+find . -type f -name *.c
```

Comment:
that's probably the best way to achieve it! :) 
but note that you're missing quotes on *.c, it should be '*.c' or "*.c".

this is due to the fact that the shell will expend the glob **before** it invokes ```find``` the same way it does with ```echo```

when you use quotes the shell will treat is as a simple string, and pass it to the find command.

you can verify it if you create a sub-directory and put a .c file inside.